### ISSUE TITLE
popover('destroy') of a jQuery cloned element fails

### ISSUE THREAD

**Issue by MSCAU** [twbs/bootstrap#19996]
This is for Bootstrap 3.3.6 on Chrome and Edge on Windows:

Starting with a DOM element (eg. el) with its standard popover, even if I **popover('destroy')** on a jQuery deep clone of it (ie. **el.clone(true, true)**), clicking the cloned element invokes the popover on the original element. I would have expected that the popover either doesn't fire at all, or fires on the clone.

See [fiddle](https://jsfiddle.net/MCAU/cxw0maks/1/)

**Comment by twbs-lmvtfy** [comment]
Hi @MSCAU!

You appear to have posted a live example (https://fiddle.jshell.net/MCAU/cxw0maks/1/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 35, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.3.4
* line 39, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.6 ; saw what appears to be usage of Bootstrap 3.3.4

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by MSCAU** [comment]
Sorry - now updated. See https://jsfiddle.net/MCAU/cxw0maks/2/

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
